NewForce-at-Mountwest/celebrity-tribute-team-marky-mark
# Hover fix on career page

I added two additional cards on the career page and fixed the photos to have standard sizing:

to test:

-hover over the card and you should see a square image that contains a shot from the movie or show with mark and a costar (or gun) There are 5 total cards.
-please check that all the headers are correct on every page
-please also check that Tommy's enlarging cards still show up
-don't forget ctrl+shift+R
